### ISSUE TITLE
Game Hangs while loading in Oculus Rift, but sound and controls work...BLACK screen in lower left hand corner!

### ISSUE THREAD

**Issue by Ar3ohdee** [KozGit/DOOM-3-BFG-VR#236]
PLEASE help, So excited to try this, but it will NOT work/load properly!!

I have tried MULTIPLE install variations with NO luck...results are always the same!

After EVERYTHING is installed and opened (ON the PC SCREEN)- The Oculus Rift page launches and the game launches. The Rift page is normal, but the game is in the lower left hand corner and is black. Sound plays through the Rift headphones/speakers and the controllers work. (In the Rift Headset)- The game acts as if it will launch and just hangs at the loading screen eventually saying "Doom 3 BFG VR is taking a long time to load, please remove headset and check screen"

Any Ideas?!? 

Win 10 
GTX 1050 Laptop 
12GB
256GB SSD
i5 6700

**Comment by Ar3ohdee** [comment]
Really, NO ONE?!??!? Is this dead in the water!?!

**Comment by Ar3ohdee** [comment]
Hello? Is there no support on this one?

**Comment by Ar3ohdee** [comment]
Can I make a donation for some help, lol

**Comment by Ar3ohdee** [comment]
FWIW:

WARNING: CVar 'developer' declared multiple times with different initial value
QA Timing INIT: 000022ms
DOOM 3 BFG VR 1.0.3.1401 win-x86 Mar  9 2017 18:56:27
2496 MHz Intel CPU with MMX & SSE & SSE2 & SSE3 & HTT
Winsock Initialized
Found interface: {91FE2AEC-FE00-43F6-ACFB-186BC3A1A50C} Realtek PCIe GBE Family Controller - 0.0.0.0 NULL netmask - skipped
Found interface: {D77BA6A7-26FA-43E9-A5D4-50E7639A7E47} Intel(R) Dual Band Wireless-AC 7265 - 192.168.43.193/255.255.255.0
Found interface: {B40FF028-B652-4B27-A4B8-BA4032260B14} Microsoft Wi-Fi Direct Virtual Adapter #2 - 0.0.0.0 NULL netmask - skipped
Sys_InitNetworking: adding loopback interface
doom using MMX & SSE for SIMD processing
enabled Flush-To-Zero mode
enabled Denormals-Are-Zero mode
------ Initializing File System ------
Current search path:
C:\Users\Ar3oh\Saved Games\id Software\DOOM 3 BFG/Fully Possessed
C:\Program Files (x86)\Steam\steamapps\common\DOOM 3 BFG Edition/Fully Possessed
C:\Users\Ar3oh\Saved Games\id Software\DOOM 3 BFG/base
C:\Program Files (x86)\Steam\steamapps\common\DOOM 3 BFG Edition/base
file system initialized.
--------------------------------------
Unknown command 'vid_restart'
Couldn't load default.cfg
Shutting down OpenGL subsystem
...wglMakeCurrent( NULL, NULL ): failed

**Comment by Ar3ohdee** [comment]
Another Log: 

WARNING: file particles/seneca.prt, line 1932: particle 'caco_lightninghit' previously defined at particles/_phrozo2_seneca.prt:1795
WARNING: file particles/seneca.prt, line 2029: particle 'caco_lightningflash' previously defined at particles/_phrozo2_seneca.prt:1961
WARNING: file particles/seneca.prt, line 2126: particle 'hellhole_endflame' previously defined at particles/_phrozo2_seneca.prt:2058
WARNING: file particles/seneca.prt, line 2178: particle 'hell_smallersparks' previously defined at particles/_phrozo2_seneca.prt:2413
WARNING: file particles/seneca.prt, line 2277: particle 'delta3_bridgesparks' previously defined at particles/_phrozo2_seneca.prt:2465
WARNING: file particles/seneca.prt, line 2330: particle 'hell1_smlavasparks' previously defined at particles/_phrozo2_seneca.prt:2564
WARNING: file particles/seneca.prt, line 2427: particle 'hell1_crushcross' previously defined at particles/_phrozo2_seneca.prt:2685
WARNING: file particles/seneca.prt, line 2506: particle 'dripsnot_colorme' previously defined at particles/_phrozo2_seneca.prt:2855
WARNING: file particles/seneca.prt, line 2610: particle 'dripdropdouble_colorme' previously defined at particles/_phrozo2_seneca.prt:2934
WARNING: file particles/seneca.prt, line 2641: particle 'hellhole_bouldersmoke' previously defined at particles/_phrozo2_seneca.prt:3038
WARNING: file particles/seneca.prt, line 2671: particle 'hellhole_boulderdirt' previously defined at particles/_phrozo2_seneca.prt:3069
WARNING: file particles/seneca.prt, line 2749: particle 'bfg_wallmark' previously defined at particles/_phrozo2_seneca.prt:3099
WARNING: file particles/seneca.prt, line 2779: particle 'BFG_CAN2' previously defined at particles/_phrozo2_seneca.prt:3177
WARNING: file particles/seneca.prt, line 2876: particle 'bfg_barrel' previously defined at particles/_phrozo2_seneca.prt:3207
WARNING: file particles/seneca.prt, line 2950: particle 'enpro_robotarmtube' previously defined at particles/_phrozo2_seneca.prt:3304
...loading 'particles/shwood.prt'
...loading 'particles/site3.prt'
...loading 'particles/smf.prt'
...loading 'particles/soulharvest.prt'
...loading 'particles/steam.prt'
...loading 'particles/ted.prt'
...loading 'particles/teleport.prt'
...loading 'particles/weapons.prt'
...loading 'particles/wormsteam.prt'
...loading 'particles/xp_cinrocket.prt'
...loading 'af/caverns2_platform.af'
...loading 'af/char_campbell.af'
...loading 'af/char_hazmat.af'
...loading 'af/char_lab1.af'
...loading 'af/char_lab2.af'
...loading 'af/char_lanternguy.af'
...loading 'af/char_maint1.af'
...loading 'af/char_marine1.af'
...loading 'af/char_npc_base.af'
...loading 'af/char_npc_misc.af'
...loading 'af/char_npcs.af'
...loading 'af/char_npcs_bag.af'
...loading 'af/char_poppy.af'
...loading 'af/char_scientist.af'
...loading 'af/char_scientist_sitting.af'
...loading 'af/char_security1.af'
...loading 'af/char_sentry.af'
...loading 'af/char_skeleton.af'
...loading 'af/char_suit1.af'
...loading 'af/char_wounded_marine.af'
...loading 'af/default.af'
...loading 'af/env_ammobelt.af'
...loading 'af/env_bluesteampipe_128.af'
...loading 'af/env_buggy.af'
...loading 'af/env_char_npcs_bag.af'
...loading 'af/env_crane.af'
...loading 'af/env_crane2.af'
...loading 'af/env_cranegame.af'
...loading 'af/env_craneplatform_caverns1.af'
...loading 'af/env_craneplatform_mcunderground.af'
...loading 'af/env_evilmeat.af'
...loading 'af/env_gibs_leftarm.af'
...loading 'af/env_gibs_leftleg.af'
...loading 'af/env_gibs_rtlolegstump.af'
...loading 'af/env_gibs_rtuplegstump.af'
...loading 'af/env_gibs_ruparmstub.af'
...loading 'af/env_gibs_spine.af'
...loading 'af/env_gibs_torso.af'
...loading 'af/env_greysteampipe_128.af'
...loading 'af/env_hellcage_long.af'
...loading 'af/env_hellcage_medium.af'
...loading 'af/env_hellcage_short.af'
...loading 'af/env_hellchain.af'
...loading 'af/env_hires_steampipe.af'
...loading 'af/env_juglamp.af'
...loading 'af/env_locust.af'
...loading 'af/env_lostsoul2_fx.af'
...loading 'af/env_marsrover.af'
...loading 'af/env_meatswing.af'
...loading 'af/env_mummy.af'
...loading 'af/env_mummy_fixed.af'
...loading 'af/env_orangesteampipe_128.af'
...loading 'af/env_pipe_ninetysix.af'
...loading 'af/env_rope.af'
...loading 'af/env_skeleton.af'
...loading 'af/env_steampipe.af'
...loading 'af/env_steampipe_a.af'
...loading 'af/env_steampipe_b.af'
...loading 'af/env_steampipe_blue_128.af'
...loading 'af/env_steampipe_c.af'
...loading 'af/env_steampipe_grey_128.af'
...loading 'af/env_steampipe_hires.af'
...loading 'af/env_steampipe_orange_128.af'
...loading 'af/env_swing_tubearm.af'
...loading 'af/env_swinglight_long.af'
...loading 'af/env_swinglight_long_fixed.af'
...loading 'af/env_swinglight_long_wbulbs.af'
...loading 'af/env_swinglight_long_wbulbs_128.af'
...loading 'af/env_swinglight_long_wbulbs_192.af'
...loading 'af/env_swinglight_long_wbulbs_32.af'
...loading 'af/env_swinglight_long_wbulbs_fixed.af'
...loading 'af/env_swinglight_round.af'
...loading 'af/env_swinglight_sl128.af'
...loading 'af/env_swinglight_sl128_fixed.af'
...loading 'af/env_swinglight_sl64.af'
...loading 'af/env_vehicle_buggy.af'
...loading 'af/env_vehicle_locust.af'
...loading 'af/env_vehicle_simplebuggy.af'
...loading 'af/env_web.af'
...loading 'af/env_wirea.af'
...loading 'af/env_wireb.af'
...loading 'af/env_wirec.af'
...loading 'af/env_xianbutton_1.af'
...loading 'af/env_xianbutton_2.af'
...loading 'af/env_xianbutton_3.af'
...loading 'af/monster_boss_cyberdemon.af'
...loading 'af/monster_boss_guardian.af'
...loading 'af/monster_boss_sabaoth.af'
...loading 'af/monster_demon_archvile.af'
...loading 'af/monster_demon_baron.af'
...loading 'af/monster_demon_cacodemon.af'
...loading 'af/monster_demon_cherub.af'
...loading 'af/monster_demon_d3xp_bruiser.af'
...loading 'af/monster_demon_hellknight.af'
...loading 'af/monster_demon_imp.af'
...loading 'af/monster_demon_lost.af'
...loading 'af/monster_demon_maggot.af'
...loading 'af/monster_demon_mancubus.af'
...loading 'af/monster_demon_mancubus2.af'
...loading 'af/monster_demon_pink.af'
...loading 'af/monster_demon_revenant.af'
...loading 'af/monster_demon_revenant2.af'
...loading 'af/monster_demon_tick.af'
...loading 'af/monster_demon_trite.af'
...loading 'af/monster_demon_vagary.af'
...loading 'af/monster_demon_vulgar.af'
...loading 'af/monster_demon_wraith.af'
...loading 'af/monster_flying_cherub.af'
...loading 'af/monster_sentry1.af'
...loading 'af/monster_tentacle.af'
...loading 'af/monster_zombie_base.af'
...loading 'af/monster_zombie_bernie.af'
...loading 'af/monster_zombie_chainsaw.af'
...loading 'af/monster_zombie_commando.af'
...loading 'af/monster_zombie_commando_cgun.af'
...loading 'af/monster_zombie_fat.af'
...loading 'af/monster_zombie_maint.af'
...loading 'af/monster_zombie_misc.af'
...loading 'af/monster_zombie_morgue.af'
...loading 'af/monster_zombie_sawyer.af'
...loading 'af/monster_zombie_scientist1.af'
...loading 'af/monster_zombie_security_machinegun.af'
...loading 'af/monster_zombie_security_pistol.af'
...loading 'af/monster_zombie_security_shotgun.af'
...loading 'af/monster_zombie_zfem.af'
...loading 'af/monster_zombie_zfem2.af'
...loading 'af/monster_zsec.af'
...loading 'af/npc.af'
...loading 'af/npczombies.af'
...loading 'af/player_doommarine.af'
...loading 'af/player_npcs.af'
...loading 'af/underground_platform.af'
...loading 'af/wounded_marine.af'
...loading 'af/zombies.af'
...loading 'newpdas/admin_banks.pda'
...loading 'newpdas/admin_dorweiler.pda'
...loading 'newpdas/admin_moses.pda'
...loading 'newpdas/admin_simons.pda'
...loading 'newpdas/alphalabs1_berger.pda'
...loading 'newpdas/alphalabs1_krietman.pda'
...loading 'newpdas/alphalabs1_lipsitz.pda'
...loading 'newpdas/alphalabs1_smith.pda'
...loading 'newpdas/alphalabs2_chin.pda'
...loading 'newpdas/alphalabs2_connors.pda'
...loading 'newpdas/alphalabs3_abrams.pda'
...loading 'newpdas/alphalabs3_lamia.pda'
...loading 'newpdas/alphalabs3_nelson.pda'
...loading 'newpdas/alphalabs3_poota.pda'
...loading 'newpdas/alphalabs4_kaczynski.pda'
...loading 'newpdas/arcade.pda'
...loading 'newpdas/caverns1_cody.pda'
...loading 'newpdas/comm1_blake.pda'
...loading 'newpdas/comm1_finch.pda'
...loading 'newpdas/comm1_wolfe.pda'
...loading 'newpdas/commoutside_holiday.pda'
...loading 'newpdas/commoutside_ridge.pda'
...loading 'newpdas/cpu_bates.pda'
...loading 'newpdas/cpu_haskell.pda'
...loading 'newpdas/cpuboss_tooloose.pda'
...loading 'newpdas/default.pda'
...loading 'newpdas/delta1_mora.pda'
...loading 'newpdas/delta1_price.pda'
...loading 'newpdas/delta2a_cinders.pda'
...loading 'newpdas/delta2a_raleigh.pda'
...loading 'newpdas/delta2a_wilson.pda'
...loading 'newpdas/delta2b_bullman.pda'
...loading 'newpdas/delta2b_erikson.pda'
...loading 'newpdas/delta2b_mcneil.pda'
...loading 'newpdas/delta2b_stemmons.pda'
...loading 'newpdas/delta3_cerano.pda'
...loading 'newpdas/delta3_lee.pda'
...loading 'newpdas/delta3_shultz.pda'
...loading 'newpdas/delta4_gilbert.pda'
...loading 'newpdas/delta5_jackson.pda'
...loading 'newpdas/delta5_swann.pda'
...loading 'newpdas/deltax_hook.pda'
...loading 'newpdas/enpro_chasar.pda'
...loading 'newpdas/enpro_hammer.pda'
...loading 'newpdas/enpro_raad.pda'
...loading 'newpdas/erebus2_gregory.pda'
...loading 'newpdas/erebus2_tanner.pda'
...loading 'newpdas/erebus2_weston.pda'
...loading 'newpdas/erebus3_crane.pda'
...loading 'newpdas/erebus3_glenn.pda'
...loading 'newpdas/erebus3_jackson.pda'
...loading 'newpdas/erebus3_kharkov.pda'
...loading 'newpdas/erebus4_mac.pda'
...loading 'newpdas/erebus4_nathan.pda'
...loading 'newpdas/erebus4_ron.pda'
...loading 'newpdas/erebus5_avery.pda'
...loading 'newpdas/erebus5_gibson.pda'
...loading 'newpdas/erebus6_grossman.pda'
...loading 'newpdas/hell_garlick.pda'
...loading 'newpdas/hellhole_id.pda'
...loading 'newpdas/info_emails.pda'
...loading 'newpdas/le_enpro_rhodes.pda'
...loading 'newpdas/le_enpro_vargas.pda'
...loading 'newpdas/le_enpro_voss.pda'
...loading 'newpdas/le_exis_adil.pda'
...loading 'newpdas/le_exis_folsom.pda'
...loading 'newpdas/le_exis_lebedev.pda'
...loading 'newpdas/le_exis_tooley.pda'
...loading 'newpdas/le_undgrnd_anders.pda'
...loading 'newpdas/le_undgrnd_thompson.pda'
...loading 'newpdas/marscity2_caseon.pda'
...loading 'newpdas/marscity2_duncan.pda'
...loading 'newpdas/marscity2_stanton.pda'
...loading 'newpdas/marscity2_tyson.pda'
...loading 'newpdas/mc1_berneche.pda'
...loading 'newpdas/mcunderground_baston.pda'
...loading 'newpdas/mcunderground_delahue.pda'
...loading 'newpdas/mcunderground_ryan.pda'
...loading 'newpdas/mcunderground_young.pda'
...loading 'newpdas/monorail_cullen.pda'
...loading 'newpdas/monorail_harding.pda'
...loading 'newpdas/monorail_hollies.pda'
...loading 'newpdas/monorail_ross.pda'
...loading 'newpdas/personal.pda'
...loading 'newpdas/phobos1_fresko.pda'
...loading 'newpdas/phobos2_edens.pda'
...loading 'newpdas/phobos2_killian.pda'
...loading 'newpdas/phobos2_noordzy.pda'
...loading 'newpdas/phobos2_owens.pda'
...loading 'newpdas/phobos3_haag.pda'
...loading 'newpdas/phobos4_jolson.pda'
...loading 'newpdas/recycling1_garza.pda'
...loading 'newpdas/recycling1_sadowayj.pda'
...loading 'newpdas/recycling2_johnson.pda'
...loading 'newpdas/recycling2_moen.pda'
...loading 'newpdas/site3_davis.pda'
...loading 'newpdas/site3_rogers.pda'
...loading 'newpdas/videos.pda'
Initializing event system
...542 event definitions
Initializing class hierarchy
...158 classes, 251488 bytes for event callbacks
Initializing scripts
--------- Game Map Shutdown ----------
--------------------------------------
********************
ERROR: Error: file script\phrozo\util.script, line 110: "bool" is not a type


********************
Error during initialization
Shutting down OpenGL subsystem
...wglMakeCurrent( NULL, NULL ): success
...deleting GL context: success
...releasing DC: success
...destroying window
...restoring hardware gamma: success

**Comment by sycspysycspy** [comment]
Did you have any other mod installed other than this?

**Comment by Ar3ohdee** [comment]
Nope

**Comment by Ar3ohdee** [comment]
ALL fresh installs today (a few times, lol)...Tried the "old" version and the new, SAME thing?!

**Comment by sycspysycspy** [comment]
Do you have steam VR installed?

**Comment by Ar3ohdee** [comment]
FIXED!

I can't EXACTLY pin point the issue as I did quite a few things after sleeping on it!!!

BUT, here is some of what I did...

I went into the integrated Intel Graphics Options and set everything to "let application decide" & and GTX 1050 (instead of "integrated"), and I think THIS is what did it ultimately. 

I also made sure to select Rift in the Audio option and microphone option in the options on the Oculus software. I also disabled/turned off Malewarebytes, CCleaner, and Blutooth?!
